Note: the codes were adopted according to the configuration environment of the Life Science Server of University of Vienna.

The [assembly] directory contains codes for trio binning, haploid genome assembly. The (pseudo)commands for executing assembly and polishing are also shwon in the directory.

The [annotation] directory contains codes for repeat annotations, transcriptome assembly (Trinity), gene model training, full-length RNA sequencing data processing, BUSCO evaluation, gene model predictions (MAKER), gene model polishing (PASApipeline), etc. 

The [Hi-C_analysese] directory contains codes for Hi-C scaffolding, Hi-C heatmap visualization, A/B compartment calling, and Perl scripts for calculating the frequency of inter-chromosomal interactions. 

The [ChIP-seq] contains codes for mapping and processing ChIP-seq data, and the [methylation] contains codes for calling 5mC DNA methylation using the Nanopore long reads.

The [genomic_feature] direcotory contains codes for comparative genomics analyses, including calcalating GC content and sequencing depth, whole-genome alignment, and mapping short- and long-reads against the reference.

The [ortholog] directory contains codes for identifying orthologous groups using OrthoFinder, and an R script for ploting the igraph figures visualising the frequency of orthologous gene pair between chicken and amphioxus chromosomes.



[assembly]: https://github.com/lurebgi/chicken-T2T/tree/main/assembly
[annotation]: https://github.com/lurebgi/chicken-T2T/tree/main/annotation
[Hi-C_analysese]: https://github.com/lurebgi/chicken-T2T/tree/main/Hi-C_analysese
[ChIP-seq]: https://github.com/lurebgi/chicken-T2T/tree/main/ChIP-seq
[methylation]: https://github.com/lurebgi/chicken-T2T/tree/main/methylation
